consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt

```diff
@@ -137,7 +144,7 @@ class KafkaEventSink(private val config: StreamsConfig,
                         }
                         TimeUnit.SECONDS.toMillis(1)
                     } else {
-                        val timeMillis = TimeUnit.MINUTES.toMillis(5)
```

Comment:
5 or 3 minutes both sound quite long, or?

Reply:
is that configureable?

Reply:
I'll add the configuration in the next PR where I'll add all the changes made in #344